components/org.wso2.carbon.datasource.core/src/test/java/org/wso2/carbon/datasource/api/DataSourceManagementServiceTest.java

```diff
@@ -27,6 +27,9 @@
 import java.net.MalformedURLException;
 import java.util.List;
 
+/**
+ * Test class for {@code DataSourceManagementService}.
```

Comment:
@code is used for denoting a block of code, it might be better to use @link:
http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#link